tox.ini

```diff
@@ -153,7 +153,8 @@ commands =
 	     --remove \
 	     requirements/base.txt \
 	     requirements/platform.txt
-         pip-compile-multi -g base -g tox -g test
+         # --allow-unsafe is a future default
```

Comment:
I'm not sure what this comment is meant to help me understand.

Reply:
It is explaining why we are passing something an option with `unsafe` in the name.